config/crd/bases/psmdb.percona.com_perconaservermongodbbackups.yaml

```diff
@@ -54,6 +58,8 @@ spec:
             type: object
           spec:
             properties:
+              backupType:
```

Comment:
I agree with this, and maybe @egegunes simply missed it since in `PerconaServerMongoDBBackupSpec` struct I can see it like this:
```
Type             pbm.BackupType           `json:"backupType,omitempty"`
```

Reply:
actually I didn't miss it, `type` just felt too generic and I went with `backupType`. If you guys think `type` is better, I'm OK with it. @hors @spron-in what do you think?